dev-python/pyjsparser/pyjsparser-2.7.1.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Fast javascript parser based on esprima.js"
+HOMEPAGE="
+	https://github.com/PiotrDabkowski/pyjsparser/
+	https://pypi.org/project/pyjsparser/
+"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
```

Comment:
I see that at least the Github repo has test cases included. Could you perhaps make a snapshot release out of the latest commit?

https://gitweb.gentoo.org/repo/gentoo.git/tree/app-editors/kakoune/kakoune-2020.01.16_p20200601.ebuild Here's an example how to do it. Name it `pyjsparser-2.7.2_p20190422.ebuild`